pantalaimon/daemon.py

```diff
@@ -1187,6 +1187,40 @@ async def _load_media(self, server_name, media_id, file_name):
         return response, decrypted_file
 
 
+    async def profile(self, request):
+        access_token = self.get_access_token(request)
+
+        if not access_token:
+            return self._missing_token
+
+        client = await self._find_client(access_token)
+        if not client:
+            return self._unknown_token
+
+        room_id = request.match_info["room_id"]
```

Comment:
This doesn't seem right? Updating a profile isn't scoped to a room.

This method mostly seems to be a copy of the `send_message` method but profiles are never uploaded encrypted and are not scoped to a room.

Has this been tested with a client? I don't see how this wouldn't end up having an encrypted avatar_url.

Reply:
I haven't tested it. it makes sense for the method not to be scoped to a room
so prifle media are alwasy unecrypted, are they? I cannot find any hint in the specs

Reply:
Yeah I would say so. Always reuploading as unencrypted seems the way to go.